app/views/posts/show.html.erb

```diff
@@ -29,9 +29,15 @@ padding: 15px;">
       <li><i class="fab fa-facebook" style="font-size: 25px;"></i></li>
     </ul>
   </div>
+
+  <h3>Comments</h3>
+  <div id="comments">
+    <%= render partial: "comments/form", locals: {commentable: @post} %>
+    <%= render @post.comments.where(parent_id: nil) %>
+  </div>
 </div>
 
-<% if current_user && current_user.id == @post.user_id %>
+<% if current_user.id == @post.user_id %>
```

Comment:
Why dont we take `@post = current_user.posts.find(id)` in controller itself so that we dont have to check `current_user.id == @post.user_id`?